### DISCUSSION TITLE
Debugging Blazor AOT RuntimeError: indirect call signature mismatch

### DISCUSSION THREAD

**Discussion by lbargaoanu** [dotnet/runtime#54049] [Q&A]
I get this error while trying to run my Blazor AOT compiled code (it works without AOT) and I'm not sure how to investigate. Can I set some flag to get a useful stack trace or some other debugging information?
[Repro](https://github.com/lbargaoanu/RoslynInBlazor/tree/CoreWf).

**Comment by danroth27** [comment]
Thank for reporting this issue! I've opened https://github.com/dotnet/runtime/issues/54264 to track it.
